CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to FHIR Converter
+
+This document describes guidelines for contributing to the FHIR Converter repo.
+
+## Submitting Pull Requests
+
+- **DO** submit all changes via pull requests (PRs). They will be reviewed and potentially be merged by maintainers after a peer review that includes at least one of the team members.
+- **DO** give PRs short but descriptive names.
+- **DO** write a useful but brief description of what the PR is for.
+- **DO** refer to any relevant issues and use [keywords](https://help.github.com/articles/closing-issues-using-keywords/) that automatically close issues when the PR is merged.
+- **DO** ensure each commit successfully builds. The entire PR must pass all checks before it will be merged.
+- **DO** address PR feedback in additional commits instead of amending.
+- **DO** assume that [Squash and Merge](https://blog.github.com/2016-04-01-squash-your-commits/) will be used to merge the commits unless specifically requested otherwise.
+- **DO NOT** submit "work in progress" PRs. A PR should only be submitted when it is considered ready for review.
+- **DO NOT** mix independent and unrelated changes in one PR.
+
+## Coding Style
```

Comment:
This coding style info is for .net. I think we can remove this section for now unless @LTA-Thinking has any recommendation.

Reply:
Our coding style is enforced by the eslint rules we have. We could mention that contributors should run the linter before submitting a PR.

Reply:
linter is already run as part of checks (which is mentioned in PR submission guidelines), so we should be good with removing this section.

Reply:
Removed the entire coding style section